app/components/component-docs/dataDisplay/CoreChip.docs.js

```diff
@@ -199,24 +209,37 @@ Chips with the onDelete prop defined will display a delete icon which changes ap
 
       <CodeSample
         title="Chip adornments"
-        description="You can add ornaments to the beginning of the component.
-Use the avatar prop to add an avatar or use the icon prop to add an icon."
-        code={`
-<CoreStack direction="row" spacing={1}>
-  <CoreChip avatar={<CoreAvatar>M</CoreAvatar>} label="Avatar" />
+        description={
+          <>
+            <CoreTypographyBody1>
+          You can add ornaments to the beginning of the component.
+Use the avatar prop to add an avatar or use the icon prop to add an icon.
+            </CoreTypographyBody1>
+
+            <CoreH6 styleClasses={[]}>Avatar Chip</CoreH6>
```

Comment:
Empty styleClasses

Reply:
I suggest adding lint checks for these to the @wrappid/core-team